Lombiq.HelpfulLibraries.Common/Utilities/CultureHelpers.cs

```diff
@@ -11,16 +11,22 @@ public static class CultureHelpers
     /// Gets all available <see cref="Country"/> objects and orders them by <see cref="Country.DisplayText"/>.
     /// </summary>
     public static IEnumerable<Country> GetCountries() =>
-        CultureInfo.GetCultures(CultureTypes.SpecificCultures).Select(culture =>
+        CultureInfo
+            .GetCultures(CultureTypes.SpecificCultures)
+            .Select(culture =>
             {
-                var regionInfo = new RegionInfo(culture.LCID); // #spell-check-ignore-line
-                return new Country
-                {
-                    TwoLetterIsoCode = regionInfo.TwoLetterISORegionName,
-                    DisplayText = regionInfo.EnglishName,
-                };
+                // This sometimes throws "CultureNotFoundException: Culture is not supported." exception on Linux, or
+                // "ArgumentException: Customized cultures cannot be passed by ID, only by name." on Windows.
+                try { return new RegionInfo(culture.LCID); } // #spell-check-ignore-line
```

Comment:
Is this intentionally a one-liner so that it is easier to use `#spell-check-ignore-line` ? If yes consider this comment resolved.